pages/docs/open-source.mdx

```diff
@@ -15,9 +15,15 @@ Learn more about what's next for Langfuse on our [roadmap](/docs/roadmap).
 
 ## License
 
-Langfuse is licensed under an [MIT license](https://github.com/langfuse/langfuse/blob/main/LICENSE), with the exception of the `/ee` folder of the repository, if applicable.
+Langfuse is licensed under an MIT license, with the exception of the `/ee` and `/web/src/ee` folders of the repository. These contain features that are available within the Enterprise Edition of Langfuse and on Langfuse Cloud. See [License](https://github.com/langfuse/langfuse/blob/main/LICENSE) for more details.
```

Comment:
These cirectories contain features that are commercially licensed. They are available on Langfuse Cloud and in the Enterprise Edition of Langfuse Self-Hosted. See.....